examples/marbles/frommarbles_flatmap.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+import rx
+from rx import operators as ops
+from rx import concurrency as ccy
+
+a = rx.cold(' ---a---a----------------a-|')
+b = rx.cold('    ---b---b---|            ')
+c = rx.cold('             ---c---c---|   ')
+d = rx.cold('                --d---d---| ')
+e1 = rx.cold('a--b--------c--d-------|   ')
+
+observableLookup = {"a": a, "b": b, "c": c, "d": d}
+
+source = e1.pipe(
+    ops.flat_map(lambda value: observableLookup[value]),
+    )
+
+source.subscribe_(
```

Comment:
Use `subscribe()` from examples now that subscribe supports (again) both observers and callbacks. Subscribe with an _ will still be used by the operators, but it may be confusing for the users that we have two different functions.

Reply:
Noted :+1: 
By the way, I've reworked all examples + some others things waiting in a big commit. I'm sorry this PR is a bit of a mess because I've pushed a lot commits that changes everything. Do you prefer I close this one and open a new one when it's ready?

Reply:
@jcafhe I think it's time to squash and merge. This feature is self contained and I'm not worried if it needs more work.

Reply:
cool !